python/ray/autoscaler/command_runner.py

```diff
@@ -164,6 +165,15 @@ def remote_shell_command_str(self) -> str:
         """Return the command the user can use to open a shell."""
         raise NotImplementedError
 
+    def run_init(self, *, as_head: bool, file_mounts: Dict[str, str]) -> None:
+        """Currently used by DockerCommandRunner to run initialization commands.
```

Comment:
nit. Can you replace  "initialization commands" with "extra initialization". I just want to make sure that it is clear that these are distinct from the YAML field `initialization_commands`.